packages/application-shell/src/components/navbar/messages.ts

```diff
@@ -5,4 +5,8 @@ export default defineMessages({
     id: 'NavBar.MCSupport.title',
     defaultMessage: 'Support',
   },
+  navBarHeader: {
+    id: 'NavBar.navBarHeader',
+    defaultMessage: 'Merchant Center',
+  },
```

Comment:
There is no need to define this as a message, the term `Merchant Center` doesn't need a translation. You can hardcode it in the component.

Reply:
Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3216/commits/a19545223dd01fad077d7c4de15e2c64222c5b0c

Thank you.

Reply:
I guess we could clean up this message now 🙂